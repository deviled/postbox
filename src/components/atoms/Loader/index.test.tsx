import React from 'react';
import { render } from '__tests__/utils';
import { Loader } from '.';

describe('Loader.tsx', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(<Loader />);

    expect(baseElement).toMatchSnapshot();
  });
});
