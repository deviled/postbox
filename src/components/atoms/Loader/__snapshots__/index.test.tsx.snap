// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader.tsx should match snapshot 1`] = `
<body>
  <div>
    <span>
      Loading ...
    </span>
  </div>
</body>
`;
